src/main/webapp/app/pages/variantRecommendation/FormatPercentage.tsx

```diff
@@ -0,0 +1,19 @@
+export function formatPercentage(value: number) {
+  if (value < 1e-4 && value > 0) {
+    let formatted = (value * 1000).toFixed(2);
+    formatted = formatted.replace(/\.?0+$/, '');
+    if (formatted.endsWith('.')) {
+      formatted = formatted.slice(0, -1);
+    }
```

Comment:
Is this to needed to remove trailing zeros? `let formatted = +(value * 1000).toFixed(2);` should give you the same result.

Reply:
I'd like to display 100% instead of 100.00%, as it looks better. However, if it doesn't matter, I can remove it.